ChangeLog

```diff
@@ -7,6 +7,9 @@ Bugfix
       MBEDTLS_THREADING_C is defined. Found by TrinityTonic, #1095
     * Fix a bug in the update function for SSL ticket keys which previously
       invalidated keys of a lifetime of less than a 1s. Fixes #1968.
+    * Fix for Clang, which was reporting a warning for the bignum.c inline
```

Comment:
Style: The alignment seems to be off by one space compared to the original and all the previous entries.

Reply:
I think that was pre-existing in the branch I based it on, but fixed anyway!